### PR TITLE
add content to the about me section

### PR DESCRIPTION
**Pull request by gratielazburlea**

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
   </head>
   <body>
-    <!-- Change this file -->
-    <h1>Hi! How are you today?</h1>
-    <p>I am the file <code>index.html</code> and you want to change me!</p>
+    <div class="despre-mine">
+      <h1>Gratiela Zburlea</h1>
+      <h2>Junior Web Developer</h2>
+      <h3>About me</h3>
+      <div class="cutie-cu-umbra">
```
Comment (marianzburlea) [inline web/index.html]: It's impressive to add custom name classes instead of copying them from the tutorial!
Keep it up!

File: web/index.html [hunk 2]
```diff
@@ -2,10 +2,22 @@
...
+        <p class="descriere-text">
+          I started my first day as a web developer on 22 December 2019
+        </p>
+      </div>
+      <button>download my resume</button>
+      <button class="disc"><i class="icoana__sageata-jos">remove me</i></button>
```
Comment (marianzburlea) [inline web/index.html]: uu, custom name classes again :)

Comment (marianzburlea) [reply web/index.html]: Any updates?

File: web/index.html (hunk 1 above)
Comment (biancalina) [reply web/index.html]: I like this class name :D

**Pull request merged**